src/widgets/Tabs/PatientSelect.js

```diff
@@ -139,9 +139,17 @@ const PatientSelectComponent = ({
   const [isQrModalOpen, toggleQrModal] = useToggle();
 
   const onQrCodeRead = ({ data }) => {
-    // TODO: Some validation might be good here but don't know format..
+    // TODO: Some validation/data sanitisation might be required here but don't know format yet...
```

Comment:
I'm not sure that it is necessary - it either matches the patient or doesn't 🤷‍♂️

Reply:
Hm, mayyybe, was a bit unsure on this. I was thinking to provide some feedback if a user tried to scan unrelated/potentially huge QR codes instead of passing them through to our patient search API 😆 , but would probably do as part of #4072